data/Dockerfiles/dovecot/decode2text.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
+
+# Example attachment decoder script. The attachment comes from stdin, and
+# the script is expected to output UTF-8 data to stdout. (If the output isn't
+# UTF-8, everything except valid UTF-8 sequences are dropped from it.)
+
+# The attachment decoding is enabled by setting:
+#
+# plugin {
+#   fts_decoder = decode2text
+# }
+# service decode2text {
+#   executable = script /usr/local/libexec/dovecot/decode2text.sh
+#   user = dovecot
+#   unix_listener decode2text {
+#     mode = 0666
+#   }
+# }
```

Comment:
I would suggest to drop the attachment extraction script. We didn‘t have attachment indexing enabled with Solr either and I don’t think we have the necessary packages installed in this image for PDF or doc/xls/ppt support. Attachments also tend to be much larger than the email message themselves and indexing would thus increase the xapian database size significatnly.